mod/worker.php

```diff
@@ -17,6 +17,9 @@ function worker_init()
 		return;
 	}
 
+	// Ensure that all "strtotime" operations do run timezone independent
+	date_default_timezone_set('UTC');
```

Comment:
I don't understand the purpose of this call, there's no `strtotime` call in this function?

Reply:
This does call some worker functionality, which then is called ```strtotime```.

BTW: For the future we should implement some ```DateTimeFormat::strToTimeUTC(...)``` function that would assume UTC as default if there is no timezone present.